pyproject.toml

```diff
@@ -13,9 +13,11 @@ dependencies = [
         "llama-index-embeddings-azure-openai==0.1.6",
```

Comment:
@gaya3-zipstack can you update the README.md as well with whatever versions have changed? For exampe, I think we missed to update the `qdrant` vector DB upgrade to 1.8.0

Reply:
@chandrasekharan-zipstack @nehabagdia @hari-kuriakose Should we maintain this version? Because now llama-index is offering bundled llm/embedding/vectordb dependencies like llama-index-llms-palm. So we will need to dig into this package to see which version of Palm is being pulled. Does it make sense to maintain versions here for the adapters here then?

Reply:
Since we no longer specify in the README.md - resolving this comment.
Only point to note regarding the importance of the README.md is that it will be displayed in PyPI